go/teams/transactions.go

```diff
@@ -582,13 +587,42 @@ func (tx *AddMemberTx) AddOrInviteMemberByAssertionOrEmail(ctx context.Context,
 	return tx.AddOrInviteMemberByUPKV2(ctx, upak, single, doInvite, assertion, role, botSettings)
 }
 
-func (tx *AddMemberTx) CompleteInviteByID(ctx context.Context, inviteID keybase1.TeamInviteID, uv keybase1.UserVersion) error {
+func (tx *AddMemberTx) ConsumeInviteByID(ctx context.Context, g *libkb.GlobalContext, inviteID keybase1.TeamInviteID, uv keybase1.UserVersion) error {
```

Comment:
I wonder if there should be an argument to this function like `expectUseInvite` which would by false by anything that doesn't expect newStyle invites (so all of sbs handling etc. currently in master). And if `expectUseInvite` is false and we go to `if isNewStyle {` branch, error out immediately

Reply:
hm, why though?

Reply:
on second thought, maybe it doesn't matter

Reply:
pre-check should save us no matter how confused the handler gets